### PR TITLE
Release 14.0.x merge master conflict resolution

### PR DESCRIPTION
**Pull request by vpnguye2**
**Summary:** Summary of changes

Addresses [CUMULUS-3135: Release Cumulus v14.0.0 ](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3135)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by npauzenga** [comment]
It looks like this was approved but there are additional conflicts that need to be resolved? Should we move this to WIP?

**Comment by vpnguye2** [comment]
Merge conflicts are resolved and ready for final review. Thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -146,17 +125,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
 ## [v13.4.0] 2022-10-31
 
-
 ### Notable changes
 
 - **CUMULUS-3104**
   - Published new tag [`43` of `cumuluss/async-operation` to Docker Hub](https://hub.docker.com/layers/cumuluss/async-operation/43/images/sha256-5f989c7d45db3dde87c88c553182d1e4e250a1e09af691a84ff6aa683088b948?context=explore) which was built with node:14.19.3-buster.
 
+### Added
```
Comment (npauzenga) [inline CHANGELOG.md:146]: So this is a change to the 13.4.0 release notes. It looks right. The notable change seems like it was 3104 and the rest of this should just be "added". Just want to verify that this was the intention.

File: CHANGELOG.md [hunk 2]
```diff
@@ -192,22 +180,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
-- **CUMULUS-3070**
```
Comment (npauzenga) [inline CHANGELOG.md:195]: Is this intentional? These 3070 changes should not be a part of 13.4.0?

File: CHANGELOG.md [hunk 3]
```diff
@@ -234,22 +206,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Pinned typescript to ~4.7.x to address typing incompatibility issues
     discussed in https://github.com/knex/knex/pull/5279
   - Update generate-ts-build-cache script to always install root project dependencies
-
-### Fixed
```
Comment (npauzenga) [inline CHANGELOG.md:238]: Same as above, just making sure these are intentional changes as they're out of scope for the 14.0 release.

File: bamboo/bootstrap-integration-tests.sh [hunk 4]
```diff
@@ -24,7 +24,8 @@ if [[ $USE_TERRAFORM_ZIPS == true ]]; then
 
   ## [MHS, 04/29/2021] fix cumulus_distribution package not released separately.
   sed -i "s/source[ ]*= \"..\/..\/tf-modules\/cumulus_distribution\"/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus.zip\/\/tf-modules\/cumulus_distribution\"/g" *.tf
-
+  # Update to use distribution module
```
Comment (npauzenga) [inline bamboo/bootstrap-integration-tests.sh:27]: Where did this come from? Not saying it's wrong but I wouldn't expect code changes in the merge from the release branch to main.

**Review by npauzenga — COMMENTED** [review]
Looks good @vpnguye2! I had a couple questions but nothing major. Mostly just making sure the changelog entries are all set.

File: bamboo/bootstrap-integration-tests.sh (hunk 4 above)
Comment (vpnguye2) [reply bamboo/bootstrap-integration-tests.sh:27]: The update is to fix CI integration tests failure when being run against the distribution module with `USE_TERRAFORM_ZIPS=true` on the Release branch.
This is also mentioned in the Changed section of the Release Note.

File: CHANGELOG.md (hunk 1 above)
Comment (vpnguye2) [reply CHANGELOG.md:146]: This section has been there with a few updates in this additional PR #[3188](https://github.com/nasa/cumulus/pull/3188) from @jennyhliu  as to help sorting out the correct order of things in this release CHANGELOG. Hence, I would assume it's meant to be there.

File: CHANGELOG.md (hunk 2 above)
Comment (vpnguye2) [reply CHANGELOG.md:195]: Correct. 3070 here in 13.4.0 was intentionally moved to 14.0.0 Changed section.

File: CHANGELOG.md (hunk 3 above)
Comment (vpnguye2) [reply CHANGELOG.md:238]: Correct in terms of this section was being restored to what should have been in [13.4.0](https://github.com/nasa/cumulus/releases/tag/v13.4.0)

**Pull request merged**